source/frontend/src/features/acquisition/list/AcquisitionFilter/AcquisitionFilter.tsx

```diff
@@ -59,10 +55,31 @@ export const AcquisitionFilter: React.FC<React.PropsWithChildren<IAcquisitionFil
     .getByType(ACQUISITION_FILE_STATUS_TYPES)
     .map(c => mapLookupCode(c));
 
+  const {
+    getAllAcquisitionFileTeamMembers: { response: team, execute: loadAcquisitionTeam },
+  } = useAcquisitionProvider();
+
+  useEffect(() => {
+    loadAcquisitionTeam();
+  }, [loadAcquisitionTeam]);
```

Comment:
nit: I'd be inclined to pass the acquisition team members as a prop to this component to make it more testable. Would prefer that the parent component (AcquisitionListView) is the one that calls the API to fetch the team members instead of the filter

Reply:
I like it